runelite-mixins/src/main/java/net/runelite/mixins/RSRegionMixin.java

```diff
@@ -77,13 +77,15 @@
 		}
 	}
 
-	@Copy("addItemPile")
-	abstract public void rs$addItemPile(int plane, int x, int y, int hash, Renderable var5, int var6, Renderable var7, Renderable var8);
+	@Copy("addGroundItemPile")
+	abstract public void rs$addGroundItemPile(int plane, int x, int y, int hash, Renderable var5, int var6, Renderable
```

Comment:
Looks like the parameter got separated from its type here

Reply:
Oh, oops, Idea autowrapping is sometimes dumb :d